inc/dropins/file-based-page-cache-functions.php

```diff
@@ -68,8 +68,13 @@ function sc_cache( $buffer, $flags ) {
 
 	$modified_time = time(); // Make sure modified time is consistent
 
+	if ( ! WP_DEBUG ) {
```

Comment:
Need to check if `WP_DEBUG` is defined properly:

`if ( ! defined( 'WP_DEBUG' ) || ! WP_DEBUG ) {`